iac-extensions/terraform/src/main/java/org/sonar/iac/terraform/checks/gcp/GcpWeakSSLProtocolCheckPart.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * SonarQube IaC Plugin
+ * Copyright (C) 2021-2022 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.iac.terraform.checks.gcp;
+
+import org.sonar.iac.terraform.checks.AbstractNewResourceCheck;
+import org.sonar.iac.terraform.symbols.AttributeSymbol;
+
+import static org.sonar.iac.terraform.checks.WeakSSLProtocolCheck.OMITTING_WEAK_SSL_MESSAGE;
+import static org.sonar.iac.terraform.checks.WeakSSLProtocolCheck.WEAK_SSL_MESSAGE;
+import static org.sonar.iac.terraform.checks.utils.ExpressionPredicate.notEqualTo;
+
+public class GcpWeakSSLProtocolCheckPart extends AbstractNewResourceCheck {
+
+  @Override
+  protected void registerResourceConsumer() {
+    register("google_compute_ssl_policy",
+      resource -> {
+        AttributeSymbol minTlsVersion = resource.attribute("min_tls_version");
+        minTlsVersion.reportIf(notEqualTo("TLS_1_2"), WEAK_SSL_MESSAGE);
```

Comment:
Should we raise if this is equal to [`TLS_1_3`](https://en.wikipedia.org/wiki/Transport_Layer_Security#TLS_1.3)? I know that's [not a possible value](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_ssl_policy), at least currently. That would look like a FP to me.
I know that's what specified in [SONARIAC-345](https://jira.sonarsource.com/browse/SONARIAC-345).